src/components/LocationPage/utils.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { Fragment } from 'react';
+import moment from 'moment';
+import { maxBy } from 'lodash';
+import { Projection, Column } from 'common/models/Projection';
+import { HashLink } from 'react-router-hash-link';
+import { scrollWithOffset } from 'components/TableOfContents';
+
+const numDays = 7;
```

Comment:
Can we get a more descriptive name or else move this inside isHospitalizationsPeak()?  Maybe HOSPITALIZATIONS_PEAK_LOOKBACK_DAYS or something?